slimeworldmanager-plugin/src/main/java/com/grinderwolf/swm/plugin/commands/CommandManager.java

```diff
@@ -81,4 +81,38 @@ public boolean onCommand(CommandSender sender, Command cmd, String label, String
     public Collection<Subcommand> getCommands() {
         return commands.values();
     }
+
+    @Override
+    public List<String> onTabComplete(CommandSender sender, Command cmd, String label, String[] args) {
+        if (args.length == 1) {
+            final List<String> toReturn = new LinkedList<>();
+            final String typed = args[0].toLowerCase();
+
+            commands.forEach((name, command) -> {
```

Comment:
If I'm not asking for too much, could you do traditional foreach loops? Using the .forEach function generates a new lambda every time (once again, waste of memory). It's not much of an issue but sometimes I get obsessed with this kinds of things xD

Reply:
Added. You think that few bytes with forEach really matters? I use forEach alot to be honest. If you can PM me on the Discord, that would be awesome. Thanks!